modules/cli/cmd/devrunner/config/genesis.json

```diff
@@ -1,39 +1,38 @@
 {
-  "genesis_time": "2024-02-21T22:23:22.87689Z",
```

Comment:
Any reason this section was moved down in the genesis.json? Not a big deal, but if this file wasn't generated, I think we should keep the json keys in the same order.